src/org/robockets/steamworks/climber/Climber.java

```diff
@@ -14,7 +14,12 @@
 	 * @return current The current of the motor.
 	 */
 	public double readCurrent() {
-		return RobotMap.powerDistPanel.getCurrent(RobotMap.climberPdpPort);
+		SmartDashboard.putNumber("current LEFT", RobotMap.powerDistPanel.getCurrent(RobotMap.climberPdpPortLeft));
+		SmartDashboard.putNumber("current RIGHT", RobotMap.powerDistPanel.getCurrent(RobotMap.climberPdpPortRight));
+		
+		return (
+				Math.abs(RobotMap.powerDistPanel.getCurrent(RobotMap.climberPdpPortLeft)) +
+				Math.abs(RobotMap.powerDistPanel.getCurrent(RobotMap.climberPdpPortRight))) / 2.0;
```

Comment:
This is really gross. There should be a variable for the current values that is both written to SmartDashboard and used in the average calculation.

And yes, for the record, I did write this. I'm cut, I know.